### ISSUE TITLE
Fomatting issues while refreshing

### ISSUE THREAD

**Issue by jordanmclain** [achannarasappa/ticker#179]
**Describe the bug**
While prices and stats are updating (refresh intervals), pieces of information disappear and reappear (ticker, price, delta).  Over time the separators disappear completely.

**To Reproduce**
Happens every time I run the program regardless of window size or options.

**Expected behavior**
Information should stay on the screen during refresh intervals.

**Screenshots**
![Screen Shot 2021-06-04 at 10 53 12 AM](https://user-images.githubusercontent.com/11875923/120830366-0814af80-c524-11eb-982c-6e63589d2303.png)

**Environment (please complete the following information):**
 - OS: MacOS 11.4 - M1
 - Terminal: Mac Terminal
 - ticker Version: v4.2.0

**Comment by erikstorli** [comment]
Yes, I'm having the same problem. Reverted to 4.1.1, and that works fine 👍

**Comment by andreblanke** [comment]
The bug appears to have been introduced in 4406ae9 with the upgrade of Bubble Tea from 12.2 to 13.4, as downgrading it again fixes the issue for me

**Comment by taviso** [comment]
I'm also experiencing this issue in 4.4.4, can the issue be reopened?
